CHANGELOG.md

```diff
@@ -1492,7 +1492,7 @@ Details
 
    Restic did not support limiting the IO concurrency / number of connections for accessing
    repositories stored using the local or SFTP backends. The number of connections is now limited
-   as for other backends, and can be configured via the the `-o local.connections=2` and `-o
+   as for other backends, and can be configured via the `-o local.connections=2` and `-o
```

Comment:
This file is generated by Calens as part of the release preparation process. The typo is from restic 0.14.0 and comes originally from one of the changelog files that usually accompany every PR. If we were to change this, it should be done in three places; here, in the original file, as well as in the release information for restic 0.14.0 on GitHub.

@MichaelEischer This can technically be done, but do you think it's worth it? After all, what's a changelog without some typos :-)

Reply:
The file in question is https://github.com/restic/restic/blob/master/changelog/0.14.0_2022-08-25/pull-3475 . I guess there's nothing wrong with fixing the typo there. That will update the `CHANGELOG.md` on the next restic release.

Fixing the typo isn't that much effort, so I don't have a strong opinion here.

Reply:
removed.